packages/clangml/clangml.3.5.0/files/clangml.install

```diff
@@ -0,0 +1,3 @@
+bin: [
+  "_build/consumer/processor.native" {"clangml-processor"}
+]
```

Comment:
Do you mean I should add a cp line in the opam file and get rid of this .install file?
I would prefer, because then the package description is more centralized.